ktor-client/ktor-client-core/common/src/io/ktor/client/plugins/websocket/WebSockets.kt

```diff
@@ -185,6 +185,11 @@ public class WebSockets internal constructor(
                 }
 
                 LOGGER.trace("Receive websocket session from ${context.request.url}: $session")
+
+                if (plugin.maxFrameSize != Long.MAX_VALUE) {
```

Comment:
Why compare with `Long.MAX_VALUE` when the default value is `Int.MAX_VALUE.toLong()`? https://github.com/ktorio/ktor/blob/3cd270690ca092fd3c01f82e2655821ec587009f/ktor-client/ktor-client-core/common/src/io/ktor/client/plugins/websocket/WebSockets.kt#L117

Reply:
Good catch, will fix
```suggestion
                if (plugin.maxFrameSize != Int.MAX_VALUE.toLong()) {
```